src/main/java/uk/gov/justice/digital/delius/service/ConvictionService.java

```diff
@@ -392,11 +392,10 @@ private ProbationStatus probationStatusOf(Offender offender) {
     }
 
     private LocalDate previouslyKnownTerminationDateOf(Offender offender) {
-        if (!probationStatusOf(offender).equals(ProbationStatus.PREVIOUSLY_KNOWN)) {
+        if (probationStatusOf(offender) != ProbationStatus.PREVIOUSLY_KNOWN) {
             return null;
         }
-        return Optional.ofNullable(offender.getEvents())
-            .orElse(Collections.emptyList())
+        return offender.getEvents()
```

Comment:
pointless `ofNullable` since spring jpa guaranteed never to return `null` for collections